### PR TITLE
Release 2022-10-31

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

Branch for Release on Monday 31st of October 2022

# Changes

* Changelog

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## Proposal 89325
+* Text improvments in Maturity card.
```
Comment (peterpeterparker) [inline CHANGELOG.md:4]: typo "improvements"

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
...
+* Two new NNS Proposal topics.
+* Two new nnsFunctions.
+* Fix Android issues with CSP policy.
```
Comment (peterpeterparker) [inline CHANGELOG.md:7]: There was a backwards compatibility issue for old devices - i.e. no issue with modern up-to-date devices, so maybe following?

* Fix backwards compatibility of the CSP policy for older Android devices.

**Review by peterpeterparker — APPROVED** [review]
LGTM, minor comments

Thanks for also bumping the version for next release 👍

File: CHANGELOG.md (hunk 2 above)
Comment (lmuntaner) [reply CHANGELOG.md:7]: Thanks, better describe yes.

**Pull request merged**